src/components/Modal/Modal.tsx

```diff
@@ -39,7 +41,7 @@ export type ModalApi = {
  * Provides underlay, modal container, and header. Will disable scrolling of page under the modal.
  */
 export function Modal(props: ModalProps) {
-  const { size = "md", content, forceScrolling, api } = props;
+  const { size = "md", content, forceScrolling, api, drawHeaderBorder} = props;
```

Comment:
**nit:** You could default this to `..., drawHeaderBorder = false } = props; `. Then your below if statement doesn't need to do the double negation operator in `if(!!drawHeaderBorder).bb.bGray200.$`